src/BaseRepository.js

```diff
@@ -6,29 +6,29 @@ class BaseRepository {
         sequelizeModel,
         mapper,
         sequelize: sequelizeModel && sequelizeModel.sequelize,
-        queryInterface: sequelizeModel && sequelizeModel.sequelize.getQueryInterface()
+        queryInterface: sequelizeModel && sequelizeModel.sequelize.getQueryInterface(),
```

Comment:
Just wondering, this will not open or keep an active connection with the database, right?

Reply:
no... sequelize controls the connection pooling.